std/experimental/allocator/building_blocks/package.d

```diff
@@ -283,9 +283,9 @@ Macros:
 MYREF2 = $(REF_SHORT $1, std,experimental,allocator,building_blocks,$2)
 MYREF3 = $(REF_SHORT $1, std,experimental,allocator,$2)
 TDC = $(TDNW $(D $1)$+)
-TDC2 = $(TDNW $(D $(MYREF2 $1,$+))$(BR)$(SMALL
+TDC2 = $(TDNW $(MYREF2 $1,$+)$(BR)$(SMALL
 $(D std.experimental.allocator.building_blocks.$2)))
-TDC3 = $(TDNW $(D $(MYREF3 $1,$+))$(BR)$(SMALL
+TDC3 = $(TDNW $(MYREF3 $1,$+)$(BR)$(SMALL
```

Comment:
![image](https://cloud.githubusercontent.com/assets/4370550/23589610/63629bd4-01d0-11e7-8c5a-e7ebf0230067.png)

yeah this is supposed to look like this: http://dlang.org/phobos/std_experimental_allocator_building_blocks.html